package.json

```diff
@@ -43,6 +41,7 @@
     "@loopback/testlab": "^4.0.0-alpha.7",
     "@types/mocha": "^2.2.43",
     "mocha": "^3.5.3",
+    "ts-node": "^3.3.0",
```

Comment:
As of https://github.com/strongloop/loopback-next-extension-starter/pull/2, we are not using `ts-node`. We are transpiling both production and test code in one go. In the development workflow, developer should start `tsc` in watch mode in background - that way files in `dist` are always up to date with the typescript sources. The tests are run via regular `mocha` then.

The project setup works well for people using VS Code. IIRC, you mentioned @virkt25 that you are using a different editor/IDE. To support your preference, we should add a new npm script for running `tsc` in watch mode, e.g. `"tsc-watch": "tsc --target es2017 --outDir dist --watch"`. Then the instructions become:

1. Start the build process in background via `npm run tsc-watch`. It will run TypeScript compiler in backround, watching and recompiling files as you change them. 
2. Run `npm run vscode-test` to re-run the test suite and lint the code for both programming and style errors.

Reply:
I created a pull request to describe dev setup for people not using VS Code - see https://github.com/strongloop/loopback-next-extension-starter/pull/8.